### DISCUSSION TITLE
I got excited for a moment xd

### DISCUSSION THREAD

**Discussion by DinoAwakening** [keiwando/toonsquid#608] [General]
<img width="580" alt="image" src="https://github.com/keiwando/toonsquid/assets/120270913/6960ef72-023b-4983-b35b-6ba40387a940">

**Comment by ArtsyStar1** [comment]
Bruh moment ):

**Comment by DWrightAnimation** [comment]
Oh trust me, I know the feeling, this wait is getting ridiculous. Surely there can't be that many more kinks to work out for 2.0.

**Reply by Gottafindaname** [reply]
Im ready too!
